scripts/deploy.ts

```diff
@@ -17,14 +17,25 @@ async function main() {
   // );
   // console.log("Testnet proxy deployed to:", testnet.address);
 
+  // const staging = await upgrades.deployProxy(
+  //   TT,
+  //   ["https://staging.tableland.network/tables/"],
+  //   {
+  //     kind: "uups",
+  //   }
+  // );
+  // console.log("Staging proxy deployed to:", staging.address);
+
   const staging = await upgrades.deployProxy(
     TT,
-    ["https://staging.tableland.network/tables/"],
+    // TODO: commenting out and hardcoding these urls feels wrong.
```

Comment:
Yeh, this is totally a pain. We can defs use the dotenv package, or just process.ENV to control the host. Can you create a ticket for this to follow up?